tests/standard_to_enhanced/planned_events_translator_test.py

```diff
@@ -362,7 +361,6 @@ def test_wzdx_creator(mock_dt, mock_dt_3, _):
                         "direction": "westbound",
                         "relationship": {},
                         "description": "Between Exit 49: CO 65; Grand Mesa (5 miles east of the Palisade area) and US 6 (Debeque) from Mile Point 50 to Mile Point 60. Road closed expect delays due to bridge construction. Until May 1, 2022 at about 12:26PM MDT.",
-                        'creation_date': "2021-10-29T18:26:04Z",
```

Comment:
Are we not parsing the creation_date value anymore?

Reply:
correct. It was not actually present in the planned events messages, there is just an update date. It is an optional field, so I am removing it